portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,8 +41,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String newComment = request.getParameter("new-comment");
-    comments.add(newComment);
+    String commentText = request.getParameter("comment-text");
+    addNewComment(commentText);
     response.sendRedirect("/index.html");
   }
+
+  private void addNewComment(String commentText) {
+    comments.add(commentText); //TO-DO: Remove once comments can be loaded from Datastore
+    
+    Entity commentEntity = new Entity("Comment");
+    commentEntity.setProperty("text", commentText);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Just thinking ahead-- you'll probably want to make this an instance variable so you can use the same instance to load comments from Datastore as well which it looks like you'll be doing next, based on your comments, and I assume will be in a different function.

Reply:
That makes sense. This line is particularly long so it's nice to not have to rewrite it haha. I added that change in the last commit.